### ISSUE TITLE
Internationalization support

### ISSUE THREAD

**Issue by justinmayer** [bennylope/django-organizations#135]
Is there any reason not to add support for Django's internationalization framework to the project? Perhaps we could start with the templates. For example, I imagine the following would suffice for the `organization_detail.html` template:

```
{% extends "organizations_base.html" %}
{% load i18n %}
{% load org_tags %}
{% block content %}
<h1>{{ organization }}</h1>
<ul>
    <li><a href="{% url "organization_edit" organization.pk %}">{% trans "Edit" %}</a></li>
    <li><a href="{% url "organization_delete" organization.pk %}">{% trans "Delete" %}</a></li>
    <li><a href="{% url "organization_user_add" organization.pk %}">{% trans "Add a member" %}</a></li>
</ul>
{% organization_users organization %}
{% endblock %}
```

What do you think?

**Comment by bennylope** [comment]
> Is there any reason not to add support for Django's internationalization framework to the project? 

There's no real reason _not_ to add internationalization support to the templates. The only reason this hasn't been done is the expectation that most people will override the templates. That being said, any contribution adding this would be welcome.

**Comment by bennylope** [comment]
Closed by #184
